# vx and vy should be in m/s, and w should be in rad/s
float64 vx
float64 vy
float64 w
